_pkgdown.yml

```diff
@@ -1,6 +1,7 @@
 template:
   bootstrap: 5
 title: mrgsolve
+url: https://github.com/metrumresearchgroup/mrgsolve
```

Comment:
Hmm, isn't this supposed to be the URL of the pkgdown site? At least for [auto-linking](https://pkgdown.r-lib.org/articles/linking.html#across-packages) to work across packages, I think that the value in `_pkgdown.yml` should be `https://mrgsolve.org/docs` and then in `DESCRIPTION` the `URL` field should be

```
URL: https://mrgsolve.org/docs, https://github.com/metrumresearchgroup/mrgsolve
```

Reply:
I've been hesitant to list those urls in the package b/c I've been maintaining it and I know CRAN will complain if the url isn't reachable. Well, it's been stable for a while so will just go for it.